### ISSUE TITLE
When zooming in, the side scroll bar is a lot bigger than the bottom scroll bar. 

### ISSUE THREAD

**Issue by NGB-Was-Taken** [Animeboynz/Mihon-Backup-Viewer#16]

**Comment by BrutuZ** [comment]
Firefox doesn't have this issue btw

**Comment by Animeboynz** [comment]
Is this still an issue or should we close it?

**Comment by NGB-Was-Taken** [comment]
Fixed but with the downside of rest of scrollbar CSS not applying. Two things we can do, just close this as completed and call it a day or close it as not planned in the favor of new issue addressing the CSS.

For reference, before and after #38.
Before:
![image](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/76197326/c25d2bc1-d48b-4114-9019-cb33a69b2c8d)

After:
![image](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/76197326/ebabb5a0-bb0b-4691-8616-8741da651d5c)

**Comment by BrutuZ** [comment]
Goddamn Chromium... `thin` makes the scrollbar on Firefox look like Chrome's, then they go and pull that xD

**Comment by NGB-Was-Taken** [comment]
I say we just completely ditch `::-webkit-scrollbar`, remove the `scrollbar-width: thin;` so it's just default size then close this as completed. 
![image](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/76197326/d8562029-7043-45b5-aedf-6efcc4f00aa0)
